### PR TITLE
Partial migration of filterbox from UA repository + adapted it to new UX design

### PR DESCRIPTION
**Pull request by sbelzile-Coveo**
2017-06-12 08_26_53-filtering controls
2017-06-12 08_27_30-filtering controls
2017-06-12 08_53_02-filtering controls
2017-06-12 08_53_14-filtering controls
2017-06-12 08_53_47-form controls

### CODE REVIEW

**Comment by sroy3** [comment]
Sorry to be picky, but all pixel values should be a variable.

File: scss/controls/filtering/filters-list.scss [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+      .section {
+        display: inline-block;
+        width: 100%;
+        padding-left: 15px;
+        margin-top: 7px;
+
+        color: $medium-blue;
+        text-transform: uppercase;
+
+        i {
```
Comment (GermainBergeron) [inline scss/controls/filtering/filters-list.scss]: can you add a class instead of using `i` ?

Comment (GermainBergeron) [inline scss/controls/filtering/filters-list.scss]: Same thing at line 36

File: docs/_components/filtering/04_filters-values.html [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+                    <span><svg viewBox="0 0 20 20" class="icon arrow-left-rounded mod-lg"><path d="M11.5 4.8L7.2 9.3c-.3.4-.3.9 0 1.3l4.3 4.6c.3.4.9.4 1.2 0 .3-.4.3-.9 0-1.3L9 9.9 12.7 6c.3-.4.3-.9 0-1.3-.3-.3-.9-.3-1.2.1z"></path></svg></span>
+                </div>
+            </div>
+            <div class="popup-main-section">
+                <div class="popup-header">
+                    <label class="selected-filter filter">Browser</label>
+                </div>
+
+                <div class="popover-body">
+                    <form>
```
Comment (sbelzile-Coveo) [inline docs/_components/filtering/04_filters-values.html]: ctrl-c + ctrl-v error => removed

File: scss/controls/filtering/filters-list.scss (hunk 1 above)
Comment (sbelzile-Coveo) [inline scss/controls/filtering/filters-list.scss]: unused => removed

File: docs/_components/filtering/01_filter-picker.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
+---
+type: filter
+title: Filter Picker
+usage: This is the standard filter picker control.
+stylesheet: '/scss/controls/filter-picker.scss'
+---
+<div class="coveo-filter-picker">
```
Comment (sbelzile-Coveo) [inline docs/_components/filtering/01_filter-picker.html]: renamed `filter-expression-picker`

File: scss/controls/filtering/filters-list.scss [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+
+    input.search {
+      box-sizing: border-box;
+      width: 100%;
+      height: $filter-pop-up-search-input-height;
+    }
+
+    .selected-filter {
+      box-sizing: border-box;
+      width: 100% !important;
```
Comment (sroy3) [inline scss/controls/filtering/filters-list.scss]: There must be a way you can get rid of `!important`

Comment (sroy3) [inline scss/controls/filtering/filters-list.scss]: If you really cannot get rid of it, just add a comment.

**Pull request merged**